state-chain/pallets/cf-validator/src/lib.rs

```diff
@@ -295,6 +295,12 @@ pub mod pallet {
 	pub(super) type MaxAuthoritySetContractionPercentage<T: Config> =
 		StorageValue<_, Percent, ValueQuery>;
 
+	/// Store the list of funded accounts and whether or not they are a active bidder.
+	#[pallet::storage]
+	#[pallet::getter(fn active_bidder)]
+	pub type ActiveBidder<T: Config> =
+		StorageMap<_, Blake2_128Concat, T::AccountId, bool, ValueQuery>;
```

Comment:
We can use the Identity hasher here to gain some efficiency.

Reply:
Changed this to a BTreeSet.